Update GO_VERSION

Hi, thanks for your script. :]

```
>>> GOPATH created at /home/pi/go
>>> Installing IPFS
bin/check_go_version 1.8
fatal: go version '1.6.3' should be at least '1.8'
=> Please take a look at https://golang.org/doc/install to install or upgrade go.
mk/golang.mk:52: recipe for target 'check_go_version' failed
make: *** [check_go_version] Error 1
```

So I changed:

`GO_VERSION=1.9`

--- a/README.md
+++ b/README.md
@@ -5,8 +5,8 @@ A bare bones [IPFS](https://ipfs.io) installer for the Raspberry Pi.
 
 ### Installation
 
-Log in to your Raspberry Pi as the default user (`pi`). From any local directory, 
-clone or download this repo, `cd` into it and run the installer:
+Log in to your Raspberry Pi with an administrator account (you can use the default `pi` user for this). From any local
+directory, clone or download this repo, `cd` into it and run the installer:
 
 ```SHELL
 $ ./install
@@ -14,11 +14,17 @@ $ ./install
 
 **Notes**
 
-* Make sure your SD card's size is 16 GB or more
-* You need to be logged in as `pi` in order to correctly run the installer. Also, 
-  the `pi` user needs root privileges to run the installer (it does by default)
-* The IPFS user directory will be created at `/home/pi/.ipfs`
-* Go's version will default to 1.6.3 (latest version as of this writing)
+* Don't execute the installation script with `sudo`
+* You'll need root privileges to run the installer (the `pi` user does so by default)
+* The IPFS user directory will be created at `~/.ipfs` (eg.: `/home/pi/.ipfs`)
+* Go's version will default to 1.9 (latest version as of this writing)
+* If you are not logged in as the default user (`pi`), adjust `ipfs-daemon.service` accordingly. Assuming you're logged
+in as `janedoe`, you'll want to update the following two lines:
+
+```INI
+User=janedoe
+Environment=IPFS_PATH=/home/janedoe/.ipfs
+```
 
 ### Tested on:
 
@@ -26,18 +32,16 @@ $ ./install
 
 - [ ] RPi 0
 - [x] RPi 1
-- [ ] RPi 2
-- [ ] RPi 3
+- [x] RPi 2
+- [x] RPi 3
 
-**SD card size**:
+**Operating system**:
 
-- [ ] 8 GB
-- [ ] 16 GB
-- [ ] 32 GB
-- [x] 64 GB
+- [ ] Noobs
+- [x] Raspbian
 
 ### How to contribute
 
-For bug reports, open a new issue. For code patches, open a pull request against 
-the `development` branch.
+All contributions are welcome. For bug reports, open a new issue. For code patches, open a pull request against the
+`development` branch.
 

--- a/install
+++ b/install
@@ -29,10 +29,10 @@ sudo apt-get -y install \
     nodejs \
     git
 
-# Maybe install Go from source (IPFS requires Go 1.5+)
+# Maybe install Go from source (IPFS requires Go 1.8+)
 GOROOT_PARENT=/usr/local
 if [ ! -x "$(command -v go)" ]; then
-    GO_VERSION=1.6.3
+    GO_VERSION=1.9
     echo ">>> Installing Go $GO_VERSION"
     GO_PKG="go$GO_VERSION.linux-armv6l.tar.gz"
     if [ ! -f $GO_PKG ]; then
@@ -45,7 +45,7 @@ else
     echo ">>> Go alredy installed. Version: $CURRENT_GO_VERSION"
     if [[ -n "$CURRENT_GO_VERSION" ]]; then
         IFS='.' read -r -a CURRENT_GO_VERSION_ARRAY <<< $CURRENT_GO_VERSION
-        if [ "${CURRENT_GO_VERSION_ARRAY[0]}""${CURRENT_GO_VERSION_ARRAY[1]}" -lt "15" ]; then
+        if [ "${CURRENT_GO_VERSION_ARRAY[0]}""${CURRENT_GO_VERSION_ARRAY[1]}" -lt "18" ]; then
             echo ">>> Current Go version is incompatible with IPFS. Please uninstall Go and run this script again."
             exit 0
         fi
@@ -80,9 +80,11 @@ fi
 # Install IPFS
 echo ">>> Installing IPFS"
 GO_EXEC=$GOROOT/bin/go
-$GO_EXEC get -d github.com/ipfs/go-ipfs
+$GO_EXEC get -u -d github.com/ipfs/go-ipfs
 cd $GOPATH/src/github.com/ipfs/go-ipfs
 make install
+cd -
+sudo cp ~/go/bin/ipfs /usr/local/bin/ipfs
 
 # Initialize IPFS
 IPFS_EXEC=$GOPATH/bin/ipfs

--- a/ipfs-daemon.service
+++ b/ipfs-daemon.service
@@ -9,7 +9,7 @@ After=network.target
 Type=simple
 User=pi
 Environment=IPFS_PATH=/home/pi/.ipfs
-ExecStart=/home/pi/go/bin/ipfs daemon
+ExecStart=/usr/local/bin/ipfs daemon
 ExecStop=/usr/bin/pkill -f ipfs
 Restart=on-failure
 RestartSec=10s

